orion/eval/evaluation.py

```diff
@@ -0,0 +1,241 @@
+import numpy as np
+from sklearn import metrics
+
+
+class Evaluator:
```

Comment:
I don't think a class is needed here, since there is nothing from the instance itself that needs to be used other than functions.

I would remove the classes and simply make them functions (which use other functions).

More specifically, I would organize this subpackage as follows:

```
orion/evaluation
├── __init__.py  -> Import the end-user functions from contextual and point
├── common.py -> Shared functions used in the contextual and point modules, including _accuracy, ...
├── contextual.py -> Import common functions and define get_contextual_confusion_matrix and contextual_accuracy, ...
├── point.py -> Import common functions and define get_point_confusion_matrix and point_accuracy...
└── utils.py
```